{"name": "test_TC_005_01_view_details_for_product[locked_out_user-secret_sauce]", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('self', <FixtureRequest for <Function test_TC_005_01_view_details_for_product[locked_out_user-secret_sauce]>>)", "trace": "file C:\\GIT\\SauceDemoProject\\tests\\test_TC_005_01_view_details_for_product.py, line 6\n  @pytest.mark.parametrize(\n      \"username, password\",\n      [\n          (\"standard_user\", \"secret_sauce\"),\n          (\"performance_glitch_user\", \"secret_sauce\"),\n          pytest.param(\n              \"locked_out_user\",\n              \"secret_sauce\",\n              marks=pytest.mark.xfail(raises=AssertionError),\n          ),\n          pytest.param(\n              \"problem_user\",\n              \"secret_sauce\",\n              marks=pytest.mark.xfail(raises=AssertionError),\n          ),\n      ],\n  )\n  def test_TC_005_01_view_details_for_product(self, d, username, password):\nE       fixture 'self' not found\n>       available fixtures: browser, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, d, doctest_namespace, extra, g, include_metadata_in_junit_xml, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\GIT\\SauceDemoProject\\tests\\test_TC_005_01_view_details_for_product.py:6"}, "attachments": [{"name": "stdout", "source": "c94eab1a-61cc-416e-83fa-cbfb5a032339-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'locked_out_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1669804274269, "stop": 1669804274269, "uuid": "ac6da5e5-ed09-4668-950b-3a85cb733e61", "historyId": "e0fe404ba58e05defac6b1db7388b7b4", "testCaseId": "3b35991d4bdcdee9d03ea37733eb2a06", "fullName": "tests.test_TC_005_01_view_details_for_product#test_TC_005_01_view_details_for_product", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(raises=<class 'AssertionError'>)"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_TC_005_01_view_details_for_product"}, {"name": "host", "value": "DESKTOP-DBCHJ8G"}, {"name": "thread", "value": "8408-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_TC_005_01_view_details_for_product"}]}